Dockerfile

```diff
@@ -28,6 +28,6 @@ RUN ${BUNDLE_INSTALL_CMD}
 
 COPY . .
 
-RUN RAILS_ENV=production bundle exec rails assets:precompile
+RUN ASSET_PRECOMPILATION_ONLY=true RAILS_ENV=production bundle exec rails assets:precompile
```

Comment:
What does this do?

Reply:
When using active storage's engine, booting the app requires a secret key base variable to be set and this variable is set through environment variables on production machines only. But the production mode is used to precompile assets. So instead of sliming the secret key base here, we use this variable as a flag that won't require active storage engine during precomplilation.

Reply:
Yep 😄